doc/howto/mod_auth_mellon_feide.rst

```diff
@@ -0,0 +1,175 @@
+================================================================================================================
+Authenticating with the apache plugin `mod_auth_mellon <https://github.com/latchset/mod_auth_mellon>`_ and Feide
+================================================================================================================
+
+Enabling the plugin on Debian
+=============================
+
+First check if the plugin is already installed and enabled::
+
+    $ sudo apache2ctl -M | grep mellon
+      auth_mellon_module (shared)
+
+If it is, go straight to `Configuration`_.
+
+If not:
+
+Install the plugin::
+
+            $ sudo apt install libapache2-mod-auth-mellon
+
+This should create the following files::
+
+    /etc/apache2/mods-available/auth_mellon.conf
+    /etc/apache2/mods-available/auth_mellon.load
+    /etc/apache2/mods-enabled/auth_mellon.conf
+    /etc/apache2/mods-enabled/auth_mellon.load
+
+Enable with::
+
+    $ sudo a2enmod auth_mellon
+
+Disable with::
+
+    $ sudo a2dismod auth_mellon
```

Comment:
If if creates files in `/etc/apache2/mods-enabled` on installation, then it's already enabled.  Is this really the default behavior?